src/main/java/com/google/devtools/build/lib/runtime/CommonCommandOptions.java

```diff
@@ -293,6 +293,14 @@ public String getTypeDescription() {
       help = "Writes into the specified file a pprof profile of CPU usage by all Starlark threads.")
   public String starlarkCpuProfile;
 
+  @Option(
+      name = "starlark_coverage_report",
+      defaultValue = "",
+      documentationCategory = OptionDocumentationCategory.LOGGING,
+      effectTags = {OptionEffectTag.BAZEL_MONITORING},
+      help = "Writes into the specified file an lcov coverage report for all Starlark files.")
```

Comment:
From a discussion on Slack: It's not immediately clear to end-users that we are using this flag to report coverage of starlark during execution of a bazel command, similar to `--starlark_cpu_profile`.

My initial impression was that this, similar to other coverage flags, are used with `bazel coverage` when testing starlark rules with Skylib's `unittest.bzl`. This is NOT the case.

Might worth being a bit more specific in this description so that Bazel's end users are clear on intended usages.

Reply:
I reworded this a little to hopefully clarify the scope. I would be interested in integrating a less low-level version of this with `bazel coverage`, but don't really know how to accomplish this yet.